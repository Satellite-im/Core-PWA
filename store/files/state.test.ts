import InitialFilesState from './state'

describe('Test files/state', () => {
  it('should match the initial state snapshot', () => {
    expect(InitialFilesState).toMatchSnapshot()
  })

  test('the function should match the initial state snapshot', () => {
    expect(InitialFilesState()).toMatchSnapshot()
  })
})
