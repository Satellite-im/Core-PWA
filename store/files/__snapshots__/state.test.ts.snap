// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test files/state should match the initial state snapshot 1`] = `[Function]`;

exports[`Test files/state the function should match the initial state snapshot 1`] = `
Object {
  "currentUpload": undefined,
  "downloadList": Array [],
  "gridLayout": true,
  "path": Array [],
  "preview": undefined,
  "rename": undefined,
  "search": Object {
    "searchAll": false,
    "value": "",
  },
  "sort": Object {
    "asc": true,
    "category": "modified",
  },
  "status": "",
}
`;
